bedrock/mozorg/templates/mozorg/moss/index.html

```diff
@@ -0,0 +1,292 @@
+        </p>
+      </div>
+    </li>
+    <li id="secure-open-source">
+      <h3 class="moss-subhead">{{ _('Secure Open Source') }}</h3>
+
+      <p class="moss-copy">
+      {% trans %}
+        The Secure Open Source (“SOS”) track supports security audits for open
+        source software projects, and remedial work to rectify the problems
+        found.
+      {% endtrans %}
+
+        {{ _('Suggest a recipient.') }}
+      </p>
+
+      <div class="moss-ctas">
+        <p>
+          <a href="{{ url('mozorg.moss.secure-open-source') }}" class="moss-copy">{{ _('Learn More') }}</a>
+        </p>
+
+        <p>
+          <a href="https://docs.google.com/a/mozilla.com/forms/d/e/1FAIpQLScLwANEOvLBE6gnFVoiamqHOYzzkaChpdQJ7f0PlZGmfyy94w/viewform" rel="external" class="button">{{ _('Nominate Now') }}</a>
+        </p>
+      </div>
+    </li>
+  </ul>
+</section>
+
+<section class="moss-section">
+  <h2 class="moss-section-heading">{{ _('A few details') }}</h2>
+
+  <p class="moss-copy">
+    {{ _('MOSS applications are considered in batches every three months.') }}
+
+  {% trans %}
+    Missing a deadline may cause a delay in the consideration of your
+    application.
+  {% endtrans %}
+
+  {% trans url1='https://wiki.mozilla.org/MOSS#Application_Deadlines' %}
+    Information on the next deadline can be found
+    <a href="{{ url1 }}">on the MOSS wiki</a>.
+  {% endtrans %}
+  </p>
+</section>
+
+<section class="moss-section">
+  <h2 class="moss-section-heading">{{ _('Who decides?') }}</h2>
+
+  <p class="moss-copy">
+  {% trans %}
+    We have formed a selection committee to assess awards on the Foundation
+    Technology and Mission Partners tracks.
+  {% endtrans %}
+
+  {% trans %}
+    It is comprised partly of current, committed Mozillians — they bring a good
+    working knowledge of Mozilla’s day- to-day activities and how various open
+    source projects are used.
+  {% endtrans %}
+
+  {% trans %}
+    There are also senior Mozilla Alumni — they are no longer actively involved
+    with Mozilla on a day-to-day basis but have a deep understanding of our
+    project and a different/outside perspective.
+  {% endtrans %}
+
+  {% trans %}
+    Finally, there are other open source experts — they bring knowledge of the
+    role of different projects within the open source ecosystem.
+  {% endtrans %}
+  </p>
+
+  <p class="moss-ctas">
+    <a href="https://wiki.mozilla.org/MOSS#Selection_Committee" rel="external" class="moss-copy">{{ _('Meet the committee') }}</a>
+  </p>
+</section>
+
+<section class="moss-section">
+  <h2 class="moss-section-heading">{{ _('Past Recipients') }}</h2>
+
+  <p class="moss-copy">
+  {% trans %}
+    Check out a few of the Mozilla Open Source Support past recipients.
+  {% endtrans %}
+  </p>
+
+  <ul class="moss-list moss-list-thirds">
+    <li>
+      {{ high_res_img('mozorg/moss/tor.png', {'alt': 'Tor', 'width': '300', 'height': '169'}) }}
```

Comment:
If this image with its alt text is acting as the title for the content, it should probably be wrapped in an `h3`. Then the dollar amount below is more content rather than a title so I'd probably just make it a paragraph (which could still use the `moss-subhead` class for styling).